be/src/storage/sstable/table.cpp

```diff
@@ -287,4 +286,9 @@ Status Table::MultiGet(const ReadOptions& options, size_t n, const Slice* keys,
     return s;
 }
 
+template Status Table::MultiGet<std::set<size_t>::iterator>(const ReadOptions& options, const Slice* keys,
```

Comment:
leave a comment on this. so if there is new container that needs to be initialized in future, it can be added the same way here.